FilterConditionEnum.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterCondition{
+   GreaterThan="greater than",
```

Comment:
Avoid spaces in enums. See how we use enums in the first spark projects

Reply:
Yes..I checked the enums in first spark project..I have removed the spaces in FilterCondition enum